### DISCUSSION TITLE
Jamstack Discord?

### DISCUSSION THREAD

**Discussion by ajcwebdev** [jamstack/jamstack.org#637] [General]
There has been a community Slack for a while, but we have seen a large shift towards Discord for developer communities. In my own experience I've found Discord a better fit mainly for two reasons:
* Discord tends to have a lower barrier of entry compared to Slack both for creating a Discord account and for joining a Discord server.
* More of the discussion is visible to the entire community at once because it isn't sectioned off into individual opt-in channels with varied numbers of people. You can still opt out of channels in Discord if you want.

The big selling point for Slack in the past was threads and Discord now has a better version of threads than Slack. In my mind the big advantage for Slack is shared workspaces and I'm not sure if that's even being used in the Jamstack Slack.

**Comment by binamkayastha** [comment]
Another point for discord: Discord has free history retention
I like searching through history before asking questions to avoid repeat discussions. It's like a growing knowledge base, the longer the community is around, the more valuable the knowledge base becomes.

The concerns for moving to discord would be potentially segmenting the community. 6K users have joined slack (though it'd be great if we could get active members somehow to get a realistic size of the community).
Maybe we could use a channel bridging tool (something like Matrix) at least between the main channels (#community and #general) to make it feel less like splitting the community?

**Comment by domitriusclark** [comment]
Coming soon to a Discord near you 🤫 👀

**Comment by ajcwebdev** [comment]
![200](https://user-images.githubusercontent.com/12433465/146072746-5fa3ebb8-330a-466a-b9cf-7af5f3be0daa.gif)
